v8.js

```diff
@@ -36,9 +36,14 @@ for (const file of files) {
     if (source.includes ("--allow-natives-syntax")) continue
     // exclude tests that use V8 extensions
     if (source.includes ("--expose-externalize-string")) continue
-    const env =
-        source.match(/environment variables:.*TZ=(?<TZ>[\S]+)/i)?.groups
-    print(`=== ${file}`)
+    let env = {}, envstr = "";
+    for (let s of source.matchAll(/environment variables:(.+)/ig)) {
+        for (let m of s[1].matchAll(/\s*([\S]+)=([\S]+)/g)) {
+            env[m[1]] = m[2];
+            envstr += ` ${m[1]}=${m[2]}`;
+        }
+    }
+    print(`=== ${file}${envstr}`);
```

Comment:
```suggestion
    let env = {}, envstr = ""
    for (let s of source.matchAll(/environment variables:(.+)/ig)) {
        for (let m of s[1].matchAll(/\s*([\S]+)=([\S]+)/g)) {
            env[m[1]] = m[2]
            envstr += ` ${m[1]}=${m[2]}`
        }
    }
    print(`=== ${file}${envstr}`)
```
No semicolons in my beautiful pristine JS code, please :-)

Reply:
OK for consistency, but not my style :)